www/js/coolpoints-util.js

```diff
@@ -72,9 +80,11 @@ function animatePointUpdate(ptEl, msgEl, diff) {
   const isPositive = diff > 0;
   const bounceAnimationName = isPositive ? "cpBounce" : "cpShake";
   const fadeAnimationName = isPositive ? "cpFadeGreen" : "cpFadeRed";
+  const glowAnimationName = isPositive ? "cpGlowGreen" : "cpGlowRed";
   const msgText = isPositive ? `+${diff}` : `${diff}`;
-
+  const btnEl = $("#cpPromptBtn");
```

Comment:
nit: in the spirit of this function, mind passing this as an arg instead? I think i use this same function for updating the table when you update user's points. In that scenario we wouldn't have a button to update or a different button to update

Reply:
Yeah that's fine, I believe I was trying to make sure it was being targeted specifically to test the functionality.